src/components/common/LargeButton.js

```diff
@@ -0,0 +1,38 @@
+// 3rd party libraries - core
+import React from 'react';
```

Comment:
Since I had to do a large refactor for the RNE upgrade anyway, it was a good time to make reusable components from our two most used buttons: the `LargeButton` which we use in several places in the app, and the `NavbarButton` which we use in most navigation screens.